cmd/bootstrap/cmd/clusters.go

```diff
@@ -35,6 +36,11 @@ func constructClusterAssignment(partnerNodes, internalNodes []model.NodeInfo, se
 		assignments[i%len(assignments)] = append(assignments[i%len(assignments)], node.NodeID)
 	}
 
+	// in place sort to order the assignment in canonical order
```

Comment:
This was to address [this issue](https://github.com/onflow/flow-go/pull/2101/files#diff-5b3a0bc04121cae2c296f5ca77bedfcd7e6ac33f441ddbcaa2291ea6003d89d1R77)

In order to ensure the identities are sorted in canonical order, I decided to sort it in the source, so that the clusters will have the sorted identities list, and no need to sort on every call to `ClusterByChainID`

Reply:
⚠️  We will also need to apply this sort when incorporating the service event [here](https://github.com/onflow/flow-go/blob/bd910749460f48d77823af422bf8baeb798b2b13/model/convert/service_event.go#L178)